### DISCUSSION TITLE
H801 misbehaving after tasmota 11 upgrade

### DISCUSSION THREAD

**Discussion by atxbyea** [arendst/Tasmota#15145] [Q&A] [answered]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._

After upgrading to tasmota 11 H801 randomly fall off WiFi, does not happen to any other devices (blitzwolf shp6, wifi led strip controller, other plugs)

Happens to both my H801, but no other devices
Was not an issue with 9.2-9.3-10.1

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [ ] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [ ] Device used (e.g., Sonoff Basic): _____
- [ ] Tasmota binary firmware version number used: _____
  - [x] Pre-compiled
  - [ ] Self-compiled
- [ ] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_
Flash H801 to 11

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._
Remain connected to WiFi

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by Jason2866** [comment, marked as answer]
Try `reset 3` let it finish. After it has finished take the device from the power supply and connect it again after a few seconds. This commands starts the wifi calibration and stores this new values.
Nothing will be deleted. All your settings stay.

**Reply by atxbyea** [reply]
Thanks @Jason2866 was optimistic for a while, but it still disconnected sometime during the night (12-18h after doing the steps)
